content/docs/dvclive/api-reference/index.md

```diff
@@ -7,5 +7,125 @@ can be imported regularly, for example:
 from dvclive import Live
 ```
 
-Will import the main class of the API:
-[`Live()`](/doc/dvclive/api-reference/live).
+<admon type="tip">
+
+If you use one of the supported [ML Frameworks](/doc/dvclive/ml-frameworks), you
+can jump directly to its corresponding page.
+
+</admon>
+
+## Basic Workflow
+
+### Initialize DVCLive
+
+```python
+from dvclive import Live
+
+live = Live()
+```
+
+See [`Live()`](/doc/dvclive/api-reference/live) for details.
+
+### Log data
+
+<toggle>
+<tab title="Scalars">
+
+```python
+live.log("acc", 0.9)
+```
+
+See `Live.log()`.
+
+</tab>
+<tab title="Images">
+
+```python
+img = np.ones((500, 500, 3), np.uint8)
+live.log_image("image.png", img)
+```
+
+See `Live.log_image()`.
+
+</tab>
+<tab title="Plots">
+
+```python
+y_true = [0, 0, 1, 1]
+y_pred = [0.2, 0.5, 0.3, 0.8]
+live.log_plot("roc", y_true, y_score)
+```
+
+See `Live.log_plot()`.
+
+</tab>
+</toggle>
+
+### (Optionally) Update the step number
+
+```python
+live.next_step()
+```
+
+See `Live.next_step()` and `Live.set_step()` for details.
+
+## Putting all together
+
+Joining the above snippets, you can include DVCLive in your training code:
+
+```python
+# train.py
+
+from dvclive import Live
+
+live = Live()
+
+for epoch in range(NUM_EPOCHS):
+    train_model(...)
+    metrics = evaluate_model(...)
+
+    for metric_name, value in metrics.items():
+        live.log(metric_name, value)
+
+    live.next_step()
+```
+
+## Outputs
+
+After you run your training code, you should see the following content in the
+project:
```

Comment:
I was never sure about the usage of term "output" here. It can be confusing in the content of DVC (stage `outs`). Maybe "project structure" or "resulting files" or something like that?

Reply:
Renamed to Output folder structure

Reply:
wdyt?

Reply:
Better, thanks. I still would remove "output", especially since the term is not even used [in the contents](https://dvc.org/doc/dvclive/outputs) (other than the title).